Bottom: 07b28c69531d5a8491e9ee8af6463cb6da65b91e
Top:    d89d2d7219b857ee6c47ef9ce4de925eec035f6a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-11 16:52:46 +0000

Reduce the redundant work done in node_label

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 03dd4d8..d103674 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1110,14 +1110,17 @@ label_contour (PLINE * a)
 	  continue;
 	}
 
+#if 0
       if (NODE_LABEL (cur) != UNKNWN)
 	{
+	  fprintf (stderr, "Not expected...\n");
 	  /* This might get hit if the last cross vertex labelling
 	   * pass was a bit keen, and labeled past its own edges?
 	   */
 	  label = NODE_LABEL (cur);
 	  continue;
 	}
+#endif
 
       if (label == INSIDE || label == OUTSIDE)
 	{
